shell/common/input_events_unittests.cc

```diff
@@ -0,0 +1,227 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/common/shell_test.h"
+#include "flutter/testing/testing.h"
+
+namespace flutter {
+namespace testing {
+
+using Generator = std::function<int(int)>;
+
+// Simulate n input events where the i-th one is delivered at
```

Comment:
Thanks for the concise description. Please put these in doxygen format though so these end up being searchable in the docs for the test.

Reply:
Done.